API/Program.cs

```diff
@@ -71,6 +80,82 @@
         options.SubstituteApiVersionInUrl = true;
     });
 
+builder
+    .Services.AddRateLimiter(options =>
+    {
+        RateLimitConfiguration rateLimitConfiguration = builder.Configuration.BindAndValidate<RateLimitConfiguration>(
+            RateLimitConfiguration.Position);
+
+        var defaultFixedOptions = new FixedWindowRateLimiterOptions()
+        {
+            PermitLimit = 30,
+            Window = TimeSpan.FromSeconds(60),
```

Comment:
I originally was confused by this var name, definitely rename to something like `anonymousRateLimiterOptions`. I thought you were choosing not to use the `rateLimitConfiguration` for some reason.